src/components/PopoverMenu/PopoverMenuItem.vue

```diff
@@ -71,7 +71,7 @@
 
 		<!-- If item.action is set instead, a button will be used -->
 		<button v-else-if="item.action" class="menuitem focusable" :class="{active: item.active}"
-			@click.stop.prevent="item.action">
+			@click.stop.prevent="item.action" v-bind:disabled="item.disabled">
```

Comment:
please use the shortand `:disabled` :)

Reply:
Mind to do the change? Feel free to merge this then as well and bring it in the server 👍